tests/Support/SupportCarbonTest.php

```diff
@@ -4,7 +4,7 @@
 
 use DateTime;
 use DateTimeInterface;
-use Tightenco\Collect\Support\Carbon;
+use Carbon\Carbon;
```

Comment:
These are as they are because they're directly mapped from the same test in the Illuminate codebase:

https://github.com/laravel/framework/blob/5.6/tests/Support/SupportCarbonTest.php

Reply:
All right. I will revert this.